bin/rucio

```diff
@@ -1459,10 +1460,16 @@ def move_rule(args):
     Update a rule.
     """
     client = get_client(args)
+
+    override = {}
+    if args.activity:
+        override['activity'] = None if args.activity.lower() == "none" else args.activity
```

Comment:
My apologies, I missed this part. It is not really possible to remove an activity, as the API will set it back to `User Subscriptions`. I think it might be preferable to not do this kind of special treatment of the argument.

https://github.com/rucio/rucio/blob/4a896696917bee18b49c902b289d189ede697dfc/lib/rucio/api/rule.py#L83-L84

Reply:
Oh good to know. I changed it 👍

Reply:
Thanks. Don’t forget to revert the change in the documentation.